Shared/sdk/net/bitstream.h

```diff
@@ -446,12 +446,12 @@ enum class eBitStreamVersion : unsigned short
 
     // setWaterLevel: add bIncludeWorldSeaLevel and bIncludeOutsideWorldLevel
     // 2020-11-03 0x70
-    SetWaterLevel_ChangeOutsideWorldLevel,
+    // SetWaterLevel_ChangeOutsideWorldLevel, - Reverted commit
```

Comment:
I think we can't do this, because servers with `SetWaterLevel_ChangeOutsideWorldLevel` bitstream version will send `SetWaterLevel_ChangeOutsideWorldLevel`  packets for clients with `PedEnterExit` version.